extensions/quick-access/README.md

```diff
@@ -0,0 +1,28 @@
+# Quick Access
+
+Pin directories you frequently visit, such as Downloads, Screenshot, and SaveFile directories.
+Quickly access (copy, open, display) new files added to pinned directories.
+
+This extension is equivalent to moving a folder to the Dock bar. Previously you had to use the mouse to access the folder in the Dock bar, now you can use Raycast to quickly access the pinned folder.
+
+For example, if you pin Downloads directory.
+You can use "Access Directory" command to quickly copy the image or open the file  when you download an image or file from your browser,.
+
+
+**Command**:
+
+- Pin Directory: Pin directories that can be accessed quickly.
+- Access Directory: Quick copy, open, reveal the latest files in pinned Finder directories.
+  - Copy to Clipboard
+  - Open in Default App/Open in Finder
+  - Reveal in Finder
+  - Pin Directory
```

Comment:
This action didn't work for me :-(

Reply:
Which action?
If it is "Copy File to Clipboard ", the implementation of this action uses AppleScript and accesses the file, you need to give Raycast the appropriate permissions in the system settings

Reply:
The `Pin Directory`. It shows me this toast. To be honest, I also don't know what should happen. I expected that the selcted directory would be pinned and then accessible in the dropdown but it wasn't.
![CleanShot 2022-04-22 at 12 14 48](https://user-images.githubusercontent.com/12066405/164714786-95940d03-8531-4a85-a5f7-4ce50f830b49.png)

Reply:
If you want to use the Pin Directory action, you need to select a folder or, alternatively, open a Finder window first.
If you do not select any folder or open a Finder window, but use the Pin Directory action, it will detect and indicate that No directories are pinned.

Reply:
Another situation is that if you select a folder that has already been pinned, when you pin it again, since the duplicate folder will not be added, it will also prompt that No directories are pinned.